x/gov/types/params.go

```diff
@@ -126,6 +113,9 @@ func validateTallyParams(i interface{}) error {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
 
+	if v.Quorum.IsNil() || v.Threshold.IsNil() || v.Veto.IsNil() {
+		return fmt.Errorf("invalid parameter type: %T", i)
```

Comment:
Please add more details to this error. I'm pretty sure this is where the simulation error is coming from.

Reply:
We can eliminate this condition now, as it's being allowed and handled